flow/packet-headers/ethernet.yaml

```diff
@@ -1,24 +1,40 @@
-openapi: 3.1.0
-
-info:
-  title: Ethernet traffic packet header
-  version: ^0.0.0
-
 components:
   schemas:
     Flow.Ethernet:
       description: Ethernet packet header
       type: object
       properties:
         dst:
-          $ref: './patterns.yaml#/components/schemas/Flow.Pattern'
+          x-pattern:
+            description: >-
+              Destination MAC address
+            format: hex
+            length: 48
+            default: '000000000000'
+            features: [count, metric_group]
         src:
-          $ref: './patterns.yaml#/components/schemas/Flow.Pattern'
+          x-pattern:
+            description: >-
+              Source MAC address
+            format: hex
+            length: 48
+            default: '000000000000'
+            features: [count, metric_group]
         ether_type:
-          $ref: './patterns.yaml#/components/schemas/Flow.Pattern'
+          x-pattern:
+            description: >-
+              Ethernet type
+            format: hex
```

Comment:
`integer`?

Reply:
Hex is better here, usually it's 0x0800 isn't it?  Rather than integer?